### ISSUE TITLE
2 or more cordova apps using this will *all* be called when custom URL is opened

### ISSUE THREAD

**Issue by pke** [EddyVerbruggen/Custom-URL-scheme#225]
I have 3 cordova apps installed that each have the other 2 apps url schema whitelisted.
When app (a) calls `window.open("appb://login", "_system")` it also itself receives that very URL in its handleOpenURL handler function. That happens on iOS and Android alike.

Is that the expected behaviour? I'd have thought only the app registered for a specific URL schema is opened?

**Comment by dbarn** [comment]
I think this is similar to the issue I am having:

https://github.com/EddyVerbruggen/Custom-URL-scheme/issues/226

**Comment by pke** [comment]
@dbarn Yes, but it seems the plugin author doesn't have a solution for that. It seems however, that the error is in the cordova-ios plugin, which listens for navigation events. 
https://github.com/apache/cordova-ios/search?utf8=%E2%9C%93&q=CDVPluginHandleOpenURLNotification

The IAB sends this notification to all listening plugins, and cordova-ios happens to also listen for it.
You need to check in your `handleOpenURL` if the scheme is related to your app.
